doc/rst/language/spec/types.rst

```diff
@@ -660,17 +660,17 @@ function (see :ref:`Formal_Type_Arguments`).
 The :mod:`Types` module provides many functions to query properties of
 types.
 
-The language provides :proc:`isCoercible <UtilMisc_forDocs.isCoercible>`,
-:proc:`isSubtype <UtilMisc_forDocs.isSubtype>`, and
-:proc:`isProperSubtype <UtilMisc_forDocs.isProperSubtype>` for comparing types.
+The language provides :proc:`isCoercible <Types.isCoercible>`,
+:proc:`isSubtype <Types.isSubtype>`, and
+:proc:`isProperSubtype <Types.isProperSubtype>` for comparing types.
 The normal comparison operators are also available to compare types:
 
  * ``==`` checks if two types are equivalent
  * ``!=`` checks if two types are different
  * ``<`` and ``>`` check if one type is a proper subtype of another (see
-   :proc:`< <UtilMisc_forDocs.<>`)
+   :proc:`< <Types.<>`)
  * ``<=`` and ``>=`` check if one type is a subtype of another (see
-   :proc:`<= <UtilMisc_forDocs.<=>`)
+   :proc:`<= <Types.<=>`)
```

Comment:
Is it possible to add some buffer space here? E.g. `< Types.<= >`.  I find it a little difficult to read, but I'm not certain if it's required

Reply:
Unfortunately, it does not work if I do that.